skin/frontend/base/default/securesubmit/css/hps-styles.css

```diff
@@ -337,6 +337,14 @@
 }
 
 
+.amasty_one_page_checkout {
```

Comment:
@ttnnkkrr We should prefix this class name with `securesubmit` or similar to prevent any possible collisions with other classes and stylesheets